Challenges_week3/hash_to_sym.rb

```diff
@@ -0,0 +1,4 @@
+
+#Convert hash values to symbols
+hash = { "OOP": "Basic", "DSA": "Advanced" }
+print hash.map { |k,v| [k, v.to_sym]}.to_h
```

Comment:
Should we use `transform_values!` on hash here? For now we are doing these things which are kinda excessive:
1. Convert hash to array via `map`
2. Inside `map` converting each value to symbol via `to_sym`
3. And then finally converting the array into hash via `to_h`